hw_9.js

```diff
@@ -0,0 +1,28 @@
+function getRandomInt(min, max) {
+    return Math.floor(Math.random() * (max - min)) + min;
+}
+
+
+function getPagesProp (){
+    let limitNumber = getRandomInt(1, 11);
+    let pageNumber = getRandomInt(1, 11);
+}
+
+
+function getPost() {
+    let xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function () {
+        if (this.readyState === 4 && this.status === 200) {
+            console.warn(JSON.parse(this.responseText));
+        }
+    };
+      xhttp.open("GET", "http://apistaging.theatre.pp.ua/posts.json?limit=${limitNumber}&page=${pageNumber}");
+      xhttp.send();
+}
+getPost(function(callback) {
```

Comment:
ти викликаєш `getPost` і передаєш аргументом колбек, а аргумент у тебе у функції не вказано. 
```suggestion
getPost(function(responseText) {
  console.log(JSON.parse(responseText))
}
```